packages/wallets/src/components/WalletsCarouselContent/WalletsCarouselContent.tsx

```diff
@@ -30,6 +36,80 @@ const WalletsCarouselContent: React.FC<TProps> = ({ onWalletSettled }) => {
     // for the embla "on select" callback
     // to avoid unbinding / cleaning etc, just let it use up-to-date list
     const walletsAccountsListRef = useRef(walletAccountsList);
+    const transitionNodes = useRef<HTMLElement[]>([]);
+    const transitionFactor = useRef(0);
+
+    const numberWithinRange = ({ max, min, number }: NumberWithinRangeProps): number =>
+        Math.min(Math.max(number, min), max);
+
+    // scale based on the width difference between active and inactive wallets
+    const transitionFactorScale = 1 - 24 / 28.8;
```

Comment:
same - typically its nice to put TRANSICTION_FACTOR_SCALE above component as it to be arbitrary constant

Reply:
Updated